src/theme/TypographyClasses/TyporgaphyClasses.mdx

```diff
@@ -0,0 +1,15 @@
+import {
+  Preview,
+  Story,
+  ArgsTable,
+  PRIMARY_STORY,
+} from "@storybook/addon-docs";
+import TyporgaphyClasses from "./TyporgaphyClasses.vue";
```

Comment:
typo in "typography" basically everywhere it looks like

Reply:
🤦🏼‍♀️